bundle/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/TcsControllerSim.scala

```diff
@@ -0,0 +1,51 @@
+package edu.gemini.seqexec.server
+
+import java.util.logging.{Level, Logger}
+
+import edu.gemini.seqexec.server.TaskRef._
+import edu.gemini.seqexec.server.TcsController._
+
+/**
+ * Created by jluhrs on 8/3/15.
+ */
+object TcsControllerSim extends TcsController {
+
+  val guideState = newTaskRef(GuideConfig(false, M1GuideOff, M2GuideOff))
+  val telescopeState = newTaskRef(TelescopeConfig(Offset.Null, Offset.Null, Offset.Null, 445e-9, 445e-9, 445e-9, Beam.A))
+  val guidersTrackState = newTaskRef(GuidersTrackingConfig(ProbeTrackingConfig.Parked, ProbeTrackingConfig.Parked, ProbeTrackingConfig.Parked, ProbeTrackingConfig.Parked))
+  val guidersActivityState = newTaskRef(GuidersEnabled(false, false, false, false))
+  val agState = newTaskRef(AGConfig(ScienceFoldPosition.Parked, HrwfsPickupPosition.Parked))
+  private val Log = Logger.getLogger(getClass.getName)
+
+  override def getConfig: SeqAction[TcsConfig] = for {
```

Comment:
In a non-sim version, this will go to the EPICS channels to get the state, right?